requirements.txt

```
@@ -1,7 +1,7 @@
 numpy>=1.15
 scipy>=1.5.4
 PyYAML
-scikit_learn==0.24.1
+scikit_learn>=0.24.2
```

we should use python version dependent requirements here; 
```suggestion
scikit_learn>=0.24.2 ; python_version == 3.6
scikit_learn >= 1.0 ; python_version > 3.6
```